examples/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{

```

Comment:
this file needs clean up

Reply:
@maximtop fixed in https://github.com/AdguardTeam/ecsstree/pull/3/commits/cbf35fd380863ac23d852b5e0bf305e295229956